src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.html

```diff
@@ -80,6 +82,9 @@
             <span class="invalid-feedback"
                   *ngIf="serviceForm.showError('service_id', frm, 'rgwPattern')"
                   i18n>The value does not match the pattern <strong>&lt;service_id&gt;[.&lt;realm_name&gt;.&lt;zone_name&gt;]</strong>.</span>
+            <span class="invalid-feedback"
+                  *ngIf="serviceForm.showError('service_id', frm, 'mdsPattern')"
+                  i18n>MDS service_id must match the pattern <strong>^[a-zA-Z_.-][a-zA-Z0-9_.-]*$</strong></span>
```

Comment:
Small nit:

I would prefer to have a more user-friendly string to show feedback instead of a regex pattern.
Maybe something like:

```suggestion
                  i18n>MDS service_id must start with a letter and and can contain alphanumeric characters, '.', '-' or '_'.
```

Reply:
Good point on more user-friendly string. After change, it looks like this now:
![Screenshot from 2022-07-21 14-45-45](https://user-images.githubusercontent.com/74922399/180291458-0b2173bc-2bc6-4f30-9e13-950cdc554b7a.png)

do we like this, or does it look like it could be misinterpreted as accepting apostrophes and/or commas?

Reply:
Even though I agree it can be confusing I think that would work, we are already using it in other places.